test/e2e/ingress/namespace.go

```diff
@@ -86,7 +88,94 @@ spec:
       servicePort: %d
 `, backendSvc, backendSvcPort[0])
 
-		assert.Nil(ginkgo.GinkgoT(), s.CreateResourceFromStringWithNamespace(route, "default"), "creating ApisixRoute")
-		_ = s.NewAPISIXClient().GET("/headers").WithHeader("Host", "httpbin.com").Expect().Status(http.StatusNotFound)
+			assert.Nil(ginkgo.GinkgoT(), s.CreateResourceFromStringWithNamespace(route, "default"), "creating ApisixRoute")
+			_ = s.NewAPISIXClient().GET("/headers").WithHeader("Host", "httpbin.com").Expect().Status(http.StatusNotFound)
+		})
+	})
+
+	ginkgo.Context("without namespace_selector", func() {
+		// make namespace_selector empty
+		s.DisableNamespaceSelector()
+		namespace := "second-httpbin-service-namespace"
```

Comment:
Is it enough that we create in the current namespace and default namespace?

Reply:
I think there is no difference between two namespaces and three or more namespaces in this test cases.

Reply:
Yes, there is no difference.  Point it out just because the content of the comment below doesn't match the logic.

Reply:
I have updated the content  of comment. Maybe CI need re-run.